packages/powersync/lib/src/open_factory/native/native_open_factory.dart

```diff
@@ -66,7 +67,11 @@ class PowerSyncOpenFactory extends AbstractPowerSyncOpenFactory {
     // ignore: deprecated_member_use_from_same_package
     _sqliteSetup?.setup();
 
-    enableExtension();
+    try {
+      enableExtension();
+    } on PowersyncNotReadyException catch (e) {
+      autoLogger.severe(e.message);
```

Comment:
This should probably rethrow the error, instead of continuing and failing with another error later on?